### ISSUE TITLE
Should Technique G138 be revised to not be a sufficient technique for 1.3.1?

### ISSUE THREAD

**Issue by backwardok** [w3c/wcag#3444]
Note: Maybe more of a discussion area in relation to what should or should not be considered sufficient within WCAG if the technique is not fully supported by browsers. 

[G138](https://www.w3.org/WAI/WCAG21/Techniques/general/G138) is marked as a sufficient technique to meet 1.3.1, with the example of using `<strong>` to provide emphasis. While browsers have a default style that's applied for `<strong>` elements, it doesn't seem like they're exposed programmatically in their respective accessibility trees (even if they're exposed in their DOM trees).

Here's how the different browsers are exposing `<strong>` in their a11y tree/node inspectors:
- Chrome: `StaticText`
- Firefox: `text leaf`
- Safari: No matching ARIA role
- Edge: `StaticText`

Part of this may be due to the `strong` role not being an ARIA role until ARIA 1.2, which only became a recommendation relatively recently. Perhaps browsers will apply that role to `<strong>` at some point. To some degree this is more of an issue with browser support than it is with HTML authors, but I was curious to what extent something should be identified as a sufficient technique if there isn't actual support for it for AT that interface with the a11y tree from the browser?

**Comment by awkawk** [comment]
AT currently read bold/strong, usually there is a verbosity setting as they aren't read by default, but the information is available to AT.

**Comment by fstrr** [comment]
This issue is labelled as a discussion, so we’re moving this to Discussions. There doesn’t seem to be an update to make to the documentation, but if that changes, we can move it back to the issues list.
